opensearch-operator/controllers/tls_test.go

```diff
@@ -49,9 +48,10 @@ var _ = Describe("TLS Reconciler", func() {
 				}},
 				NodePools: []opsterv1.NodePool{
 					{
-						Component: "masters",
-						Replicas:  3,
-						Roles:     []string{"master", "data"},
+						Component:   "masters",
+						Replicas:    3,
+						Roles:       []string{"master", "data"},
+						Persistence: &opsterv1.PersistenceConfig{PersistenceSource: opsterv1.PersistenceSource{EmptyDir: &corev1.EmptyDirVolumeSource{}}},
```

Comment:
thats related to the PR?

Reply:
The test was running into problems because no configured persistence means PVC and that meant the recovery code tried to kick in during the test. By changing the persistence it is ignored.